tensorflow/tools/dist_test/python/mnist_replica.py

```diff
@@ -56,9 +56,9 @@
                      "Worker task index, should be >= 0. worker_index=0 is "
                      "the master worker task the performs the variable "
                      "initialization ")
-flags.DEFINE_integer("num_workers", None,
+flags.DEFINE_integer("num_workers", 2,
                      "Total number of workers (must be >= 1)")
-flags.DEFINE_integer("num_parameter_servers", 2,
+flags.DEFINE_integer("num_parameter_servers", 1,
                      "Total number of parameter servers (must be >= 1)")
```

Comment:
These flags are redundant if you have the two lists `--ps_hosts` and `--worker_hosts`. However, if you delete them, there will be consequences for the automated tests that run this code. @caisq: Can you comment on any other changes that would be necessary?